schema/deploy/shipping/views.sql

```diff
@@ -1565,6 +1709,7 @@ create materialized view shipping.scan_encounters_v1 as
               when vac_name_1 in (values('pfizer'), ('moderna')) and vac_name_2 in (values('pfizer'), ('moderna')) and date_part('day', encountered::timestamp - vac_date_2::timestamp) >= 14 then 'fully_vaccinated'
               when vac_name_1 in (values('pfizer'), ('moderna'), ('johnson')) and date_part('day', encountered::timestamp - vac_date::timestamp) >= 1 then 'partially_vaccinated'
               when date_part('day', encountered::timestamp - vac_date::timestamp) = 0 then 'not_vaccinated'
+              when vac_name_3 in (values('pfizer'), ('moderna')) || vac_name_1 = 'johnson' and vac_name_2 (values('pfizer'), ('moderna'), ('johnson')) then 'boosted'
```

Comment:
Use `or` instead of concatenate (`||`) here. 

Also need to add in a date check. "boosted" should be 14 days after receiving the shot to be consistent with other statuses.